cmpctircd/Controllers/OperController.cs

```diff
@@ -49,12 +49,12 @@ public class OperController : ControllerBase {
                 }
                 // Instantiate the algorithm through reflection, if not already instantiated.
                 HashAlgorithm algorithm;
-                if(!_algorithms.TryGetValue(ircop.Algorithm, out algorithm))
-                    algorithm = _algorithms[ircop.Algorithm] = (ircop.Algorithm.GetConstructor(Type.EmptyTypes)?.Invoke(new object[0]) as HashAlgorithm) ?? throw new IrcErrNoOperHostException(client);
+                if(!_algorithms.TryGetValue(ircop.AlgorithmType, out algorithm))
```

Comment:
Sorry about this part. We'll figure something out.

We should test server linking is still working.